tests/integration/test_s3.py

```diff
@@ -217,7 +217,7 @@ async def test_handling_non_existing_file_and_bucket(
         )
 
     with pytest.raises(ObjectStorageProtocol.BucketNotFoundError):
-        # copy when source does not exist:
+        # copy when destination does not exist:
```

Comment:
Maybe this could be rephrased to "destination bucket"? Destination objects don't exist in most copy use cases...

Reply:
Changed in 4df451e and refined the non-existing source bucket/object case.